languages/python/reference/README.md

```diff
@@ -538,3 +538,181 @@ These will very rarely be encountered in the wild, the first because it's more o
+|  [Multiple assignment][multiple-assignment] | [ ]() | [ ]() |
+|  [Mutation][mutation-general] | [ ]() | [ ]() |
+|  [Mutability in Python][mutability]| [ ]() | [ ]() |
+| [Namespaces][namespaces]| [ ]() | [ ]() |
+| [Nested functions][nested-functions]`functional`| [ ]() | [ ]() |
+| [nonlocal][keyword-nonlocal] | [ ]() | [ ]() |
+| [Non-Public Methods][non-public-methods]| [ ]() | [ ]() |
+|  [None][keyword-none]|   [#1031](https://github.com/exercism/v3/issues/1031)         |         |
+| [not][keyword-not]| [ ]() | [ ]() | 
+| [object][builtin-types-object] `OOP`| [ ]() | [ ]() |
+| [Operators][operators] | [ ]() | [ ]() | 
+| [Operator overloading][operator-overloading] | [ ]() | [ ]() | 
+| [Operator precedence][operator-precedence] | [ ]() | [ ]() | 
+| [or][keyword-or] | [ ]() | [ ]() |
+| [Order of Evaluation][order-of-evaluation]| [ ]() | [ ]() |
+| [Partial application][partial-application] `functional`| [ ]() | [ ]() |
+| [pass][keyword-pass]| [ ]() | [ ]() |
+| [Polymorphism][polymorphism-general]| [ ]() | [ ]() |
+| [Positional parameters][positional-parameters] | [ ]() | [ ]() |
+| [Positional-only parameters][positional-only-parameters] | [ ]() | [ ]() |
+| [Powers of Two][powers-of-two]| [ ]() | [ ]() |
+| [property][builtin-types-property] `OOP`| [ ]() | [ ]() |
+| [Property Decorator][property-decorator]| [ ]() | [ ]() |
+| [raise][raise]| [ ]() | [ ]() |
+|[range][builtin-types-range] `bsic`| [ ]() | [ ]() |
+|[recursion][recursion] `functional`| [ ]() | [ ]() |
+|[Recursive data structures][recursive-data-structures] | [ ]() | [ ]() |
+|[Regular Expressions][regular-expressions] | [ ]() | [ ]() |
+|[return][keyword-return]| [ ]() | [ ]() |
+|[Return Values][return-value]| [ ]() | [ ]() |
+| [Rich comparison methods][rich-comparison-methods] | [ ]() | [ ]() | 
+|[Scope][scope]| [ ]() | [ ]() |
+| [set][builtin-types-set] `basic` | [ ]() | [ ]() | 
+|  [Set comprehension][set-comprehension] |  [ ]()  |  [ ]()  | 
+| [slice][builtin-types-slice] `advanced`| [ ]() | [ ]() |
+| [Slicing][slicing]| [ ]() | [ ]() |
+| [Short circuiting][short-circuiting]| [ ]() | [ ]() |
+|[State][state] | [ ]() | [ ]() |
+| [Static Methods][static-methods] | [ ]() | [ ]() |
+|[str][builtin-types-str] `basic` | []() | [#824](https://github.com/exercism/v3/pull/824)|
+| [String formatting][string-formatting] `basic`| [ ]() | [ ]() |
+| [String methods][string-methods] `basic`| [ ]() | [ ]() |
+| [String splitting][string-splitting] `basic`| [ ]() | [ ]() |
+| [String translation][string-translation] `basic`| [ ]() | [ ]() |
+| [True][keyword-true]| [ ]() | [ ]() |
+| [try][keyword-try]| [ ]() | [ ]() |
+|[tuple][builtin-types-tuple] `basic`| [ ]() | [ ]() |
+|[Tuple unpacking][tuple-unpacking]| [ ]() | [ ]() |
+| [type][builtin-types-type] `OOP`| [ ]() | [ ]() |
+| [type conversion][type-conversion] `basic`| [ ]() | [ ]() |
+| [Type hinting][type-hinting] | [ ]() | [ ]() |
+| [ [Variables]] [variables] | [ ]() | [ ]() |
```

Comment:
Space between [label] and [reference] is breaking rendering.

Reply:
Clearly, I need another markdown editor (I am only half-joking here).  PyCharm, WebStorm, Macdown, AND Typora have all failed me in one way or another.  So have the plugins for Atom.  Never thought I'd miss Excel (and I don't _really_) .. but editing a long table sure is handy in spreadsheet form.